OneSignalSDK/app/build.gradle

```diff
@@ -68,6 +68,8 @@ dependencies {
 
     // For Chrome tabs
     implementation 'com.android.support:customtabs:28.0.0'
+
+    implementation 'com.huawei.hms:push:4.0.3.301'
```

Comment:
Should we also add in the location dependency?
`implementation 'com.huawei.hms:location:X.X.X.X'`